clang/lib/Sema/SemaDecl.cpp

```diff
@@ -13530,9 +13530,12 @@ void Sema::AddInitializerToDecl(Decl *RealDecl, Expr *Init, bool DirectInit) {
   }
 
   if (VDecl->isInvalidDecl()) {
-    CorrectDelayedTyposInExpr(Init, VDecl);
+    ExprResult Res = CorrectDelayedTyposInExpr(Init, VDecl);
+    std::vector<Expr *> SubExprs;
```

Comment:
This can be an ArrayRef (or at the minimum a SmallVector)

Reply:
Switched to `SmallVector`.